templates/docker-compose/docker-compose.yml.dist

```diff
@@ -53,6 +56,34 @@ services:
             - db_password
             - db_root_password
 
+    supportpal_websockets:
+        container_name: '${WS_SERVICE_NAME}'
+        image: 'public.ecr.aws/supportpal/helpdesk:${APP_VERSION}-ws-buster'
+        restart: always
+        depends_on:
+            - db
+            - redis
+            - mailer
+        environment:
+            REDIS_HOST: ${CACHE_SERVICE_NAME}
+            REDIS_PORT: ${REDIS_PORT}
+            HOST: ${HOST}
+            MAILER_SERVICE_NAME: ${MAILER_SERVICE_NAME}
+            WS_SERVICE_NAME: ${WS_SERVICE_NAME}
+        volumes:
+            - supportpal_resources:/supportpal/resources
+            - supportpal_app:/supportpal/app
+            - supportpal_config:/supportpal/config/production
+            - supportpal_storage:/supportpal/storage
+            - ./customization/:/customization
+        networks:
+            - supportpal_backend
+            - supportpal_frontend
```

Comment:
Why does it need to be in `supportpal_frontend`?

Reply:
frontend network so nginx can reach it, and backend network so it can communicate with database